dbt_copilot_helper/templates/svc/manifest-public.yml

```diff
@@ -88,7 +88,7 @@ variables:                    # Pass environment variables as key value pairs.
   OTEL_TRACES_SAMPLER: traceidratio
   OTEL_TRACES_SAMPLER_ARG: "0.05"
 {%- for envvar, value in env_vars.items() %}
-  {{ envvar }}:
+  {{ envvar }}: {{ value }}
```

Comment:
I don't know how this have never been noticed before.

Reply:
I wonder if there were any tests covering it?

Reply:
They were wrong...

https://github.com/uktrade/copilot-tools/pull/294/files#diff-82d568a363d34c429ce01cb7ac84669d295b0cc8410d9b0bcee7ac47e8f7d84dR90

I'm sure it would have come to light on the next migration.